Solutions/Infoblox Cloud Data Connector/Analytic Rules/Infoblox-TI-SyslogMatchFound-URL.yaml

```diff
@@ -0,0 +1,67 @@
+id: 28ee3c2b-eb4b-44de-a71e-e462843fea72
+name: Infoblox - TI - Syslog Match Found - URL
+description: |
+  'Syslog URL match found in your Infoblox TIDE Threat Intelligence. Customize query count, scheduling, responses and more. Modify data sources, types and threat properties as desired.'
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: Syslog
+    dataTypes: 
+      - Syslog
+  - connectorId: ThreatIntelligence
+    dataTypes: 
+      - ThreatIntelligenceIndicator
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1498
+  - T1565
+query: |
+  let dt_lookBack = 1h;
+  let ioc_lookBack = 14d;
+  let TI = ThreatIntelligenceIndicator
+  | where TimeGenerated >= ago(ioc_lookBack) and ExpirationDateTime > now()
```

Comment:
Hi @sschuur to fix the failing kql validation, can you please replace the line "_**| where TimeGenerated >= ago(ioc_lookBack) and ExpirationDateTime > now()**_" in query with below lines: -

  | where TimeGenerated >= ago(ioc_lookBack)
  | summarize LatestIndicatorTime = arg_max(TimeGenerated, *) by IndicatorId
  | where Active == true and ExpirationDateTime > now()
  
  also remoce **_| where Active == true_** from two lines below this to avoid repeated filtering. Below is a screenshot of changes for your reference, thanks.
  
  
![image](https://github.com/Azure/Azure-Sentinel/assets/115772287/caa87495-9853-44d8-8e48-05a0ff75c0e2)